code/drasil-lang/Language/Drasil/Chunk/Eq.hs

```diff
@@ -28,34 +35,37 @@ instance Idea          QDefinition where getA c = getA $ c ^. qua
 instance HasSpace      QDefinition where typ = qua . typ
 instance HasSymbol     QDefinition where symbol e = symbol (e^.qua)
 instance Definition    QDefinition where defn = defn'
-instance Quantity      QDefinition where 
+instance Quantity      QDefinition where
 instance DefiningExpr  QDefinition where defnExpr = equat
 instance Eq            QDefinition where a == b = (a ^. uid) == (b ^. uid)
 instance MayHaveUnit   QDefinition where getUnit = getUnit . view qua
+instance ExprRelat     QDefinition where relat z = sy z $= z ^. equat
```

Comment:
This is definitely the correct implementation of this, but is still something we'll want to shift elsewhere in implementation.